iroh/src/magicsock/metrics.rs

```diff
@@ -30,8 +30,10 @@ pub struct Metrics {
     /*
      * Holepunching metrics
      */
-    /// The number of NAT traversal attempts initiated.
-    pub nat_traversal: Counter,
+    /// The number of remote endpoints for which of NAT traversal was initiated.
+    pub remote_holepunch_attempts: Counter,
```

Comment:
The "remote_" prefix makes me think this was initiated by the remote. I think "holepunch_attempts" is fine.